doc/documentation/src/tutorial_templates/tutorial_poisson.rst.j2

```diff
@@ -0,0 +1,205 @@
+
+When we treat the problem with the scalar transport or thermal solver, of course we also consider only the steady state situation as a special case of the equation above.
+The mesh can be generated by any pre-processor, see the :ref:`pre-processing section<preprocessing>`; in our case we created the exodus file `poisson_problem_geo.e` with Cubit.
+The mesh is shown below; node sets 1 to 4 representing the boundary lines are defined, the whole surface is node set number 5.
+
+
+.. figure:: /_assets/tutorial_scatra_mesh.png
+   :alt: tutorial_scatra mesh
+   :width: 500px
+   :align: center
+
+   2D Mesh of for the Poisson problem with node sets.
+
+
+The **boundary conditions** for either case are that the primary variable (:math:`T` or `C`, respectively) is zero at the boundary.
+Thus, we have to define a Dirichlet boundary condition at the four lines, which for both problem types is denoted identically by the lines
+
+{{ section_dump(scatra_file_content, "DESIGN LINE DIRICH CONDITIONS") }}
+
+There is a constant source term over the whole structure, that is, the right hand side of the above equation is constant.
+This is a Neumann, i.e., flux, condition, which is given by
+
+{{ section_dump(scatra_file_content, "DESIGN SURF NEUMANN CONDITIONS") }}
+
+For both problem types, we run the simulation in a single step.
+
+
+Poisson problem solved with the scalar transport solver
+--------------------------------------------------------
+
+The diffusion problem is a so-called scalar transport problem in |FOURC|, thus the ``PROBLEM TYPE`` section is denoted here by ``Scalar_Transport``;
+it defines most of the further sections in the input file und thus is of special importance:
+
+
+{{ section_dump(scatra_file_content, "PROBLEM TYPE") }}
+
+The time step, and time independence (steady state) situation are then declared in the ``SCALAR TRANSPORT DYNAMIC`` section.
+We solve the problem using a direct linear solver (``UMFPACK``, see section ``SOLVER 1``), since the problem consists of only 441 nodes is very small.
+The solver of this section is selected by the keyword ``LINEAR_SOLVER: 1``.
+The time step does not matter, since the problem is steady state, but a value for ``TIMESTEP`` must be given anyway.
+
+{{ section_dump(scatra_file_content, ["SCALAR TRANSPORT DYNAMIC", "SOLVER 1"]) }}
+
+The diffusivity, which is the only material parameter in this model, is given in the material section:
+
+{{ section_dump(scatra_file_content, "MATERIALS") }}
+
+The geometry defined in the exodus file (see above) is read in by the command
+
+{{ section_dump(scatra_file_content, "TRANSPORT GEOMETRY") }}
+
+which then in the ``ELEMENT_BLOCK`` section defines the cell type, the material and the transport formulation (in our case it is Standard).
+
+For scalar transport simulations, the primary variable is always printed into vtk files as ``phi_1``.
+The flux cannot be printed.
+Thus, no output section is necessary in this file.
```

Comment:
I don't understand this. What does `flux cannot be printed` mean? How is it related to the output section?